frontend/prebuild/src/app/login/login.component.ts

```diff
@@ -40,10 +40,12 @@ export class LoginComponent implements OnInit, OnDestroy {
     this.meta.addTag({name: 'viewport', content: `width=${viewWidth}, height=${viewHeight}, initial-scale=1.0`}, true);
 
     this.route.params.subscribe( params =>  {
-      sessionStorage.setItem("jwt", params['jwt']);
+    	  if (params['jwt'])
+        sessionStorage.setItem("jwt", params['jwt']);
     });
-    if (sessionStorage.getItem('jwt')) {
-      this.loginThroughGoogle();
+    var jwt = sessionStorage.getItem('jwt');
```

Comment:
Nitpick: should be `let` to not leak into global scope.

Reply:
I never knew that was the difference between var and let, thanks I'll change that